easybuild/easyconfigs/h/HDF5/HDF5-1.10.4-PGI-18.10-GCC-5.4.0-2.26-serial.eb

```diff
@@ -0,0 +1,34 @@
+# @author: Guilherme Peretti Pezzi and Luca Marsella (CSCS)
+
+easyblock = 'ConfigureMake'
+
+name = 'HDF5'
+version = '1.10.4'
+versionsuffix = '-serial'
```

Comment:
Ins't the HDF5 serial and the parallel version called HDF5-parallel?

Reply:
That's the naming convention used previously for `Kesch`.